libc/src/math/generic/CMakeLists.txt

```
@@ -495,6 +493,27 @@ add_entrypoint_object(
     libc.src.__support.macros.optimization
 )
 
+add_entrypoint_object(
+  sinpi
+  SRCS
+    sinpi.cpp
+  HDRS
+    ../sinpi.h
+    range_reduction_double_fma.h
```

range_reduction_double_fma.h is already part of the `range_reduction_double` header library.

```suggestion
  HDRS
    ../sinpi.h
```